examples/nbody_oscillator.py

```diff
@@ -0,0 +1,54 @@
+import taichi as ti
+
+ti.init(arch=ti.cuda) # Try to run on GPU, use ti.opengl if you don't have CUDA
+
+N = 8000
+pos = ti.var(dt=ti.f32, shape=(N, 2))
```

Comment:
A better practice is `pos = ti.Vector(2, dt=ti.f32, shape=N)`. The same thing for `vel`. Then you can make use of vector arithmetic in the `advance` function.

Reply:
Good idea! But when I have this example written, OpenGL hadn't support Matrix yet and I had to norm_sqr by hand :) Will change this trr.